.mergify.yml

```diff
@@ -69,7 +69,7 @@ pull_request_rules:
     actions:
       post_check:
         success_conditions:
-          - "title ~= ^(fix|feat|docs|style|refactor|perf|test|build|ci|chore|revert)(?:\\(.+\\))?:"
+          - "title ~= ^(fix|feat|docs|style|refactor|perf|test|build|ci|chore|revert)(?:\\(.+\\))?:|(merge\squeue:)"
```

Comment:
This is not the right fix, we should not even run this action if the head is a merge-queue branch

Reply:
`- -head~=^mergify/merge-queue/.*`